content-src/components/DiscoveryStreamBase/DiscoveryStreamBase.jsx

```diff
@@ -1,15 +1,41 @@
+import {CardGrid} from "content-src/components/DiscoveryStreamComponents/CardGrid/CardGrid";
 import {connect} from "react-redux";
+import {Hero} from "content-src/components/DiscoveryStreamComponents/Hero/Hero";
+import {HorizontalRule} from "content-src/components/DiscoveryStreamComponents/HorizontalRule/HorizontalRule";
+import {List} from "content-src/components/DiscoveryStreamComponents/List/List";
 import React from "react";
+import {SectionTitle} from "content-src/components/DiscoveryStreamComponents/SectionTitle/SectionTitle";
+import {TopSites} from "content-src/components/DiscoveryStreamComponents/TopSites/TopSites";
 
 export class _DiscoveryStreamBase extends React.PureComponent {
+  renderComponent(component) {
+    switch (component.type) {
+      case "TopSites":
+        return (<TopSites {...component.properties} />);
+      case "SectionTitle":
+        return (<SectionTitle {...component.properties} />);
+      case "CardGrid":
+        return (<CardGrid {...component.properties} />);
+      case "Hero":
+        return (<Hero {...component.properties} />);
```

Comment:
I suppose this should be fine for now especially lacking designs. If we end up needing to copy/paste a bunch or there isn't that much difference in the react components, we probably could minimize bugs in similar-but-not-quite code with something like `<PocketCards type={component.type} {...components.properties} />`

Reply:
slightly less code if you want:

```js
const ComponentTypes = {TopSites, CardGrid, Hero};
...
// in renderComponent
const ItemComponent = ComponentTypes[component.type];
return (<ItemComponent {...component.properties} />
```